package.json

```diff
@@ -32,6 +32,7 @@
     "chai": "^3.5.0",
     "d3": "^4.1.1",
     "del": "^2.2.1",
+    "deferred-js": "^1.0.2",
```

Comment:
No it's not. That was introduced when I started introducing models into the react module. I'll remove that